lib/pipeline/webpackProcess.js

```diff
@@ -3,7 +3,16 @@ const fs = require('../core/fs');
 const ProcessWrapper = require('../core/processes/processWrapper');
 const webpack = require('webpack');
 const writeFile = require('util').promisify(require('fs').writeFile);
-const {errorMessage} = require('../utils/utils');
+const {errorMessage, recursiveMerge} = require('../utils/utils');
+
+const defaultPipelineConfig = {
+  typescript: false
```

Comment:
I can see some code regarding flow: isFlowEnabled
shall you add:
`flow: false`

Reply:
Good question, I forgot to include something in the notes about that.

I'm asking for input on this, but here's why I didn't include `flow: false`:

By default `babel-preset-react-app` activates the `@babel/plugin-transform-flow-strip-types` plugin. So, to make use of Flow type checking in a create-react-app project, a developer only needs to follow these instructions: [Adding Flow](https://github.com/facebook/create-react-app/blob/master/packages/react-scripts/template/README.md#adding-flow).

So, with this PR as-is, an embark user can follow those same instructions and they will just work. However, since a DApp's `config/pipeline.js` allows TypeScript support to be activated, and since Flow and TS can't both be active at the same time, embark's webpack config toggles Flow off if TS is toggled on.

Reply:
What do you think of adding a comment in pipeline.js on how to add flow (with a link), and specify that typescript must set at false?

Reply:
Great suggestions, will do.

Reply:
Done.  Let me know if you think the comments I intro'd need revision.

Reply:
LGTM